charts/stable/standardnotes-server/values.yaml

```diff
@@ -0,0 +1,285 @@
+      localstack:
+        localstack:
+          mountPath: /etc/localstack/init/ready.d/localstack_bootstrap.sh
+          subPath: localstack_bootstrap.sh
+portal:
+  open:
+    enabled: false
+securityContext:
+  container:
+    readOnlyRootFilesystem: false
+    runAsGroup: 0
+    runAsUser: 0
+service:
+  main:
+    enabled: true
+    ports:
+      main:
+        enabled: true
+        port: 3000
+        targetPort: 3000
+      files:
+        enabled: true
+        port: 3125
+        targetPort: 3104
+  localstack:
+    enabled: true
+    targetSelector: localstack
+    ports:
+      app:
+        enabled: true
+        targetSelector: localstack
+        port: 4566
+        targetPort: 4566
+workload:
+  main:
+    podSpec:
+      containers:
+        main:
+          env:
+            DB_HOST:
+              secretKeyRef:
+                expandObjectName: false
+                name: '{{ printf "%s-%s" .Release.Name "mariadbcreds" }}'
+                key: plainhost
+            DB_PORT: 3306
+            DB_USERNAME: std_notes_user
+            DB_PASSWORD:
+              secretKeyRef:
+                expandObjectName: false
+                name: '{{ printf "%s-%s" .Release.Name "mariadbcreds" }}'
+                key: mariadb-password
+            DB_DATABASE: standard_notes_db
+            DB_TYPE: mysql
+            REDIS_PORT: 6379
+            REDIS_HOST:
+              secretKeyRef:
+                expandObjectName: false
+                name: '{{ printf "%s-%s" .Release.Name "rediscreds" }}'
+                key: plainhost
+            CACHE_TYPE: redis
+            AUTH_JWT_SECRET:
+              secretKeyRef:
+                name: standardnotes-secrets
+                key: AUTH_JWT_SECRET
+            AUTH_SERVER_ENCRYPTION_SERVER_KEY:
+              secretKeyRef:
+                name: standardnotes-secrets
+                key: AUTH_SERVER_ENCRYPTION_SERVER_KEY
+            VALET_TOKEN_SECRET:
+              secretKeyRef:
+                name: standardnotes-secrets
+                key: VALET_TOKEN_SECRET
+  localstack:
+    enabled: true
+    type: Deployment
+    podSpec:
+      containers:
+        localstack:
+          enabled: true
+          primary: true
+          imageSelector: localstackImage
+          probes:
+            liveness:
+              enabled: false
+            readiness:
+              enabled: false
+            startup:
+              enabled: false
+          env:
+            SERVICES: "sns,sqs"
+            HOSTNAME_EXTERNAL: localstack
+            LS_LOG: warn
+mariadb:
+  enabled: true
+  mariadbUsername: std_notes_user
+  mariadbDatabase: standard_notes_db
+redis:
+  enabled: true
+configmap:
+  localstack-bootstrap-init:
+    enabled: true
+    data:
+      localstack_bootstrap.sh: |-
+        #!/usr/bin/env bash
+        set -euo pipefail
+        echo "configuring sns/sqs"
+        echo "==================="
+        LOCALSTACK_HOST=localhost
+        AWS_REGION=us-east-1
```

Comment:
Can you please elabore why we hardcoding aws region here?
I bet it has something to do with the dummy id bellow.
Is it only for "startup"? (eg can the user configure this afterwards somehow?)

I wonder why such a long script is needed, isnt anything built in to do the initialization?

Reply:
I'm not familiar enough with aws or localstack to know. That script is [provided by standardnotes](https://github.com/standardnotes/server/blob/main/docker/localstack_bootstrap.sh) for use with the image. The [localstack config notes](https://docs.localstack.cloud/references/configuration/) don't show a replacement for things like creating topics.

The region appears to be a [hardcoded default](https://github.com/standardnotes/server/blob/5c9f493b67a65aa9e659db77e797635b8aa09fc3/packages/websockets/src/Bootstrap/Container.ts#L84), I'm guessing that value is set to guarantee functionality with the dummy AWS services.

Reply:
Hmm cool then.

Maybe we fetch the script from upstream at startup? What do you think?
eg https://raw.githubusercontent.com/standardnotes/server/main/docker/localstack_bootstrap.sh

Reply:
Would certainly save space. I took a pass at an initContainer in latest commit. The [docs for localstack](https://docs.localstack.cloud/references/init-hooks/) read as though this is needed on every init.

Reply:
Despite passing the github checks, I don't believe it is working. The bootstrap file is missing once the pod state reaches ready. There's probably some detail of initcontainers I don't understand yet.

Reply:
I forgot what persistence is for a second. Fixed now.